String FPD_Header;
String FPD_GPSWeek;
String FPD_GPSTime;
String Heading;
String Pitch;
String Roll;
String Lattitude;
String Longitude;
String Altitude;
String Ve;
String Vn;
String Vu;
String Baseline;
String NSV1;
String NSV2;
String Status;
String FPD_Cs;
String FPD_CrLf;